src/Symfony/Bundle/TwigBundle/Resources/config/twig.php

```diff
@@ -172,5 +173,9 @@
         ->set('controller.template_attribute_listener', TemplateAttributeListener::class)
             ->args([service('twig')])
             ->tag('kernel.event_subscriber')
+
+        ->set('twig.validator', TwigValidator::class)
+            ->args([service('twig')])
+            ->tag('validator.constraint_validator')
```

Comment:
Should be removed in the bundle extension if validator is not available

Reply:
Done